draft of 3.11 install updates

kalexand-rh opened the pull request.

kalexand-rh commented on install/running_install.adoc:
```diff
@@ -165,41 +137,35 @@ ifdef::openshift-enterprise[]
+** If your installation failed, xref:advanced-retrying-installation[retry the installation].
 
 [[running-the-advanced-installation-containerized]]
-=== Running the Containerized Installer
```
Is everything in this section still valid?

wkshi replied:
@kalexand-rh LGTM

kalexand-rh commented on install/running_install.adoc:
```diff
@@ -435,31 +402,78 @@ $ ansible-playbook --user openshift \
 
-After you resolve the errors, you can continue installation:
+////
+[IMPORTANT]
```
Is this note still valid?

kalexand-rh replied:
I put it back in. Thanks!

kalexand-rh commented on install/running_install.adoc:
```diff
@@ -435,31 +402,78 @@ $ ansible-playbook --user openshift \
+====
+////
+
+[[advanced-retrying-installation]]
```
I made a new section that contains some information that didn't fit well in the existing structure. I think having a separate section about how to retry the installer might help. Please let me know if any information needs to be added.

sdodson replied:
They were added by the console and monitoring teams. They can be added to the list or omitted I don't have a strong opinion one way or the other. If they fail then I think the error message is enough for the admin to continue without changes to the documentation.

If we want to add them I can look into the ordering and where to put them.

sdodson replied:
We add playbooks quite frequently, I don't know what criteria we should use to determine whether or not we should document them.

kalexand-rh replied:
@sdodson If they're part of the core installation process, they either need to be added to the list of playbooks at ~470 or we need to remove the table and the "run the rest of the playbooks by hand" method. I'm not sure what drove the inclusion of the "run the rest of the playbooks by hand method," so I can't say which is better.

I'm on the fence about making a table to list all the playbooks that drift in and out and whether they're GA, tech preview, or dev preview/unsupported. (I think it might be a useful addition, but we'd need to talk about a sane method for maintaining it.)

kalexand-rh commented on install/running_install.adoc:
```diff
@@ -593,117 +607,19 @@ of your etcd hosts in the following:
-----
-
-[[uninstalling-nodes-advanced]]
-=== Uninstalling Nodes
```
I made a new uninstall topic because I think that's a different use case.

kalexand-rh commented on install/running_install.adoc:
```diff
@@ -593,117 +607,19 @@ of your etcd hosts in the following:
 Configuration documentation].
 
-[[optionally-securing-builds]]
-== Optionally Securing Builds
```
I don't think this subsection made sense, so I moved it to a prereq.

kalexand-rh replied:
You're right. The section makes it sound like you do it with installation. Thank you!

kalexand-rh commented on _topic_map.yml:
```diff
@@ -265,23 +265,25 @@ Name: Installing Clusters
   File: host_preparation
 - Name: Configuring Your Inventory File
   File: configuring_inventory_file
 - Name: Example Inventory Files
```
Will you PTAL at the rendered version of the Example Inventory Files topic? I might just be confused, but I am not sure that all the scenarios are valid with etcd on static pods.

jianlinliu replied:
@gpei PTAL

gpei replied:
About the `Example Inventory Files` topic

- etcd related inventory file configuration lgtm. We still support both staic pod etcd and rpm etcd in ocp-3.11. When etcd co-located with master, then it would run as static pod on master node, or else etcd would be installed as stand-alone rpm etcd. One thing we could mention here is etcd stand-alone hosts on atomic is no longer supported. 

- Some outdated parameter in the example inventory file "Multiple Masters Using HAProxy Inventory File"
    - openshift_node_kubelet_args is deprecated, could use node config edits instead 
    - openshift_clock_enabled=true is redundant since all the places calling it were set to True by default
```
# apply updated node defaults
openshift_node_kubelet_args={'pods-per-core': ['10'], 'max-pods': ['250'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}

# enable ntp on masters to ensure proper failover
openshift_clock_enabled=true
```

kalexand-rh replied:
@gpei, if users wanted to use static etcd pods on masters, what would they need to specify in the inventory file?

Is the static pod the preferred etcd deployment method in 3.10 and 3.11?

When was `openshift_node_kubelet_args` deprecated? I see several references in the collection.

vrutkovs replied:
>When was openshift_node_kubelet_args deprecated? I see several references in the collection.

In 3.10, during the move to node group configmaps. Instead the node edits should be used:
```
openshift_node_groups=[{'name': 'node-config-all-in-one', 'labels': ['node-role.kubernetes.io/master=true', 'node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'kubeletArguments.pods-per-core','value': ['20']}]}]
```

kalexand-rh replied:
I'm making an issue to track any other openshift_node_kubelet_args deprecation changes that aren't included in this PR: https://github.com/openshift/openshift-docs/issues/12200

kalexand-rh commented:
@xltian, will you PTAL? I bumped the versions and made some text edits for style and structure edits to try to make the topics make more sense. I also had a couple of sections that I'm not sure are still valid - I added comments to those sections.

kalexand-rh commented:
[Preview build here](http://file.rdu.redhat.com/kalexand/092018/edit_install/install/index.html)

xltian commented:
@jianlinliu @jiajliu from QE will review

sdodson commented on install/configuring_inventory_file.adoc:
```diff
@@ -934,8 +934,8 @@ openshift_hosted_registry_routecertificates= "{'certfile': '<path>/org-cert.pem'
 
-If you are using a Cockpit registry console image other than the default or
-require a specific version of the console, specify the desired registry within
+If you use a Cockpit registry console image other than the default or
```
Yes, this point can be removed.

kalexand-rh commented:
@gpei, PTAL at https://github.com/openshift/openshift-docs/pull/12154/commits/083151a77c3e41aac85c2451cd9c8dab75e1edaf

kalexand-rh commented:
@sdodson, @gpei, @jianlinliu, @xltian 

@vikram-redhat asked me to go ahead and merge these changes. I think I've addressed all your feedback so far. If you have more changes or I missed something, please tag me in new comments on the PR so I can open a new one or make a bug and assign it to me.

kalexand-rh commented:
/cherrypick enterprise-3.11
